.github/workflows/pr-title-linter.yml

```diff
@@ -16,9 +16,15 @@ jobs:
     steps:
       - name: Checkout Repo
         uses: actions/checkout@v4
+
+      - name: Setup
+        id: config
+        uses: ./.github/actions/init
         with:
-          fetch-depth: 0
-          show-progress: false
+          turbo-signature: ${{ secrets.TURBO_REMOTE_CACHE_SIGNATURE_KEY }}
+          turbo-team: ${{ vars.TURBO_TEAM }}
+          turbo-token: ${{ secrets.TURBO_TOKEN }}
+          playwright-enabled: true # Must be present to enable caching on branched workflows
```

Comment:
(future thought) I feel like we could make the name for this more explicit, based on the comment.